logentries/metrics.py

```diff
@@ -639,23 +639,7 @@ def load_json(self, conf):
         # Basic metrics
         metricDict = conf.get(METRIC)
         for item in self.DEFAULTS:
-            try:
                 self.__dict__[item] = metricDict.get(PREFIX + item)
```

Comment:
Its not need for our json configuration.
Logentries will still correctly follow logs without it. 
But, shall we keep it in? I have it correctly loading the parameters with unit tests.